test/test_pullparser.rb

```diff
@@ -62,6 +62,26 @@ def test_entity_replacement
       end
     end
 
+    def test_character_references
+      source = '<a>&#65;</a><b>&#x42;</b>'
+      pp = REXML::Parsers::PullParser.new( source )
+      el_name = ''
```

Comment:
Let's use better name in new tests:

```suggestion
      parser = REXML::Parsers::PullParser.new( source )
      element_name = ''
```

Reply:
OK
I see.